node/fs/handle.js

```diff
@@ -301,6 +300,43 @@ class FileHandle extends EventEmitter {
    * @TODO
```

Comment:
not necessary anymore

Reply:
I have `TODO` there for me to come input a comment 🙏🏼